files/zh-cn/web/html/element/heading_elements/index.html

```diff
@@ -109,7 +109,7 @@ <h2 id="可访问性问题">可访问性问题</h2>
 
 <h3 id="导航">导航</h3>
 
-<p>A common navigation technique for users of screen reading software is jumping from heading to heading to quickly determine the content of the page. Because of this, it is important to not skip one or more heading levels. Doing so may create confusion, as the person navigating this way may be left wondering where the missing heading is.</p>
+<p>对于使用阅读屏幕软件的用户而言，一种常见的导航方式是从一个标题跳到另一个标题，以快速确定页面的内容。 因此，不要跳过一个或多个标题级别。因为这样做可能会造成混乱，使用户困惑于缺少的标题在哪里。</p>
```

Comment:
`阅读屏幕软件` ---> `屏幕阅读软件`：https://support.google.com/chromebook/answer/7031755?hl=zh-Hans
```suggestion
<p>对于使用屏幕阅读软件的用户而言，一种常见的导航方式是从一个标题跳到另一个标题，以快速确定页面的内容。 因此，不要跳过一个或多个标题级别。因为这样做可能会造成混乱，使用户困惑于缺少的标题在哪里。</p>
```